esmvaltool/diag_scripts/clouds/clouds_ecs_groups_maps.py

```diff
@@ -0,0 +1,516 @@
+                    calculate_bias(cube, cube_obs).data,
+                    calculate_rmsd(cube, cube_obs).data,
+                    calculate_corr(cube, cube_obs).data
+                ]
+
+            sample_stat = sample_stat.astype(float)
+            stat = sample_stat.describe()
+            basename = f"bootstrapping_{attributes['short_name']}_{group}"
+            csv_path = (get_diagnostic_filename(basename,
+                                                cfg).replace('.nc', '.csv'))
+            stat.to_csv(csv_path)
+            logger.info("Wrote %s", csv_path)
+
+
+def main(cfg):
+    """Run diagnostic."""
+    cfg = deepcopy(cfg)
+    cfg.setdefault('plot_each_model', False)
+    cfg.setdefault('plot_bias', False)
+
+    input_data = list(cfg['input_data'].values())
+
+    groups = group_metadata(input_data, 'variable_group', sort='dataset')
+    attributes = next(iter(extract_variables(cfg).values()))
+    all_groups = list(group_metadata(input_data, 'variable_group'))
+
+    # Read data
+    cubes, cubes_out = read_data(groups, cfg)
+
+    # Plotting climatologies
+    plot_diagnostic(cubes_out, attributes, input_data, cfg)
+
+    if cfg['reference']:
+        # Compute bias plots
+        cube_obs = cubes_out.extract_cube(
+            iris.Constraint(cube_func=lambda cube: cube.attributes[
+                'variable_group'] == 'OBS'))
+
+        # Bootstrapping
+        bootstrapping(cubes, cube_obs, all_groups, attributes, cfg)
+
+        # Compute statistics
+        df = get_dataframe(cubes, cube_obs)
+
+        # write statistics
+        write_statistics(df, attributes, cfg)
+
+        # compute bias
+        cubes_diff = iris.cube.CubeList()
+        attributes['short_name'] = attributes['short_name'] + "_diff"
+
+        if cfg['plot_bias']:
+            for cube in cubes_out:
+                if (cube.attributes['variable_group'] != 'OBS'
+                        or cube.attributes['dataset'] != 'MultiModelMean'):
+                    logger.info("Processing %s of group %s",
+                                cube.attributes['dataset'],
+                                cube.attributes['variable_group'])
+                    bias = calculate_bias(cube, cube_obs)
+                    rmsd = calculate_rmsd(cube, cube_obs)
+                    corr = calculate_corr(cube, cube_obs)
+                    cube_diff = cube - cube_obs
+                    cube_diff.attributes = cube.attributes
+                    cube_diff.var_name = cube.var_name
+                    cube_diff.attributes['short_name'] = attributes[
+                        'short_name']
+                    cubes_diff.append(cube_diff)
+                    logger.info('%s : bias = %f, rmsd = %f, corr = %f',
+                                cube.attributes['variable_group'], bias.data,
+                                rmsd.data, corr.data)
+
+            # Plotting biases
+            plot_diagnostic(cubes_diff, attributes, cfg)
```

Comment:
```suggestion
            plot_diagnostic(cubes_diff, attributes, input_data, cfg)
```

See https://app.codacy.com/gh/ESMValGroup/ESMValTool/pull-requests/3526/issues#issue-b363dc06aeb74b23ccbd25d260e3d182

Reply:
Thanks for spotting!